velox/dwio/common/BufferedInput.h

```diff
@@ -17,30 +17,33 @@
 #pragma once
 
 #include "velox/dwio/common/DataBuffer.h"
+#include "velox/dwio/common/Options.h"
 #include "velox/dwio/common/SeekableInputStream.h"
 #include "velox/dwio/common/StreamIdentifier.h"
 
 namespace facebook::velox::dwio::common {
 
 class BufferedInput {
  public:
-  constexpr static uint64_t kMaxMergeDistance = 1024 * 1024 * 1.25;
-
   BufferedInput(
       std::shared_ptr<ReadFile> readFile,
       memory::MemoryPool& pool,
+      folly::Executor* FOLLY_NULLABLE executor = nullptr,
```

Comment:
We no longer use FOLLY_NULLABLE. Please, remove.

Reply:
Please put it as last parameter.  Also after you move the main functionality into `ReadFile`, you no longer need it here.